.github/workflows/azure-static-web-apps-orange-flower-0da9ed11e.yml

```diff
@@ -1,61 +1,56 @@
-name: Azure Static Web Apps CI/CD
-
```

Comment:
Adding env:
  API_URL: ${{secrets.yourSecretName}}
here like done in class will allow you to simplify the url entered in the axios call